include/deal.II/meshworker/integration_info.templates.h

```diff
@@ -191,32 +191,32 @@ namespace MeshWorker
     face_flags |= boundary_flags;
     neighbor_flags |= neighbor_geometry ? boundary_flags : update_default;
 
-    if (cell_selector.has_values())
+    if (cell_selector.has_values() != 0)
```

Comment:
I just noticed this one - before I looked for `0u`. It looks like old versions of deal.II had the `!= 0` check which converts the left side to an integer when its already a boolean - odd. I'll fix it.